src/app/settings/settings.component.ts

```diff
@@ -239,4 +248,27 @@ export class SettingsComponent extends RouterEnter {
     return true;
   }
 
+  async uploadProfileImage(file, type = null) {
+    if (file.success) {
+      this.imageUpdating = true;
+      this.settingService.updateProfileImage({
+        image: file.data.url
+        }).subscribe(
+        success => {
+          this.imageUpdating = false;
+          this.profile.image = file.data.url;
+          const casheUser = this.storage.getUser();
```

Comment:
Don't need to get the whole user object, it is handled inside the function already, check how other places use `setUser()`